ducksteroid/Test/RankingTests.cs

```diff
@@ -0,0 +1,128 @@
+using NUnit.Framework;
+using System.Collections.Generic;
+using System;
+using MyGame.src;
+
+namespace Tests
+{
+    public class RankingTests
+    {
+        //set-up method to return a ranking obj with 4 scores in it
+        public Ranking Setup()
+        {
+            Ranking rank = new Ranking();
+            Score score1 = new Score(20);
+            Score score2 = new Score(15);
+            Score score3 = new Score(2);
+            Score score4 = new Score(15);
+            rank.AddScore(score1);
+            rank.AddScore(score2);
+            rank.AddScore(score3);
+            rank.AddScore(score4);
+            return rank;
+        }
+
+        //test: check if ranking adds the correct score
+        //expected: contain the correct score reference (true)
+        [Test]
+        public void TestAddCorrectScore()
+        {
+            Ranking rank = Setup();
```

Comment:
We shouldn't have AddScore in SetUp because it will make this test not useful at all.